indico/modules/events/models/persons.py

```diff
@@ -317,7 +317,7 @@ def person(cls):
                 cls.person_link_backref_name,
                 cascade='all, delete-orphan',
                 cascade_backrefs=False,
-                lazy='dynamic'
```

Comment:
needs merge_person_info to be adapted (maybe also add the missing session block person link in there)